starter-code/src/clue.js

```diff
@@ -1,94 +1,106 @@
-last_name:    Peacock
-color:        blue
-description:  She is from a wealthy family and uses her status and money to earn popularity
-age:          36
-image:        https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg
-occupation:   Socialité
+var missScarlet= {
+first_name:   "Kasandra",
+last_name:    "Scarlet",
+color:        "red",
+description:  "She is an A-list movie star with a dark past",
+age:          31,
+image:        "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg" ,
+occupation:   "Actor",
+}
 
-mrMustard
-first_name:   Jack
-last_name:    Mustard
-color:        yellow
-description:  He is a former football player who tries to get by on his former glory
-age:          62
-image:        https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg
-occupation:   Retired Football player
+var mrsPeacock={
+first_name:   "Eleanor",
+last_name:    "Peacock",
+color:        "blue",
+description:  "She is from a wealthy family and uses her status and money to earn popularity",
+age:          36,
+image:        "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg" ,
+occupation:   "Socialité",
+}
 
+var mrMustard=
+{first_name:  " Jack",
+last_name:   " Mustard",
+color:        "yellow",
+description:  "He is a former football player who tries to get by on his former glory",
+age:          62,
+image:        "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+occupation:   "Retired Football player",
+}
 // Weapons
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+var  rope = {name: "rope", weight: 10}
+var knife = {name : "knife", weight: 8}
+var candlestick = {name :"candlestick", weight: 2}
+var dumbbel = {name :"dumbbell", weight: 30}
+var poison = {name : "poison", weight: 2}
+var axe = {name : "axe", weight: 15}
+var bat = {name : "bat", weight: 13}
+var trophy = {name : "trophy", weight: 25}
+var pistol = {name : "pistol", weight: 20}
 
 // Rooms
-name: Dinning Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+var dinningRoom ={name: "Dinning Room"}
+var Conservatory ={name: "Conservatory"}
+var Kitchen= {name: "Kitchen"}
+var Study= {name: "Study"}
+var Library ={name: "Library"}
+var BilliardRoom= {name:"Billiard Room"}
+var Lounge ={name: "Lounge"}
+var Ballroom ={name: "Ballroom"}
+var Hall ={name:" Hall"}
+var Spa ={name: "Spa"}
+var  LivingRoom ={name: "Living Room"}
+var Observatory ={name: "Observatory"}
+var Theater ={name: "Theater"}
+var  GuestHouse= {name: "Guest House"}
+var Patio ={name: "Patio"}
```

Comment:
this works fne but be careful, by convention var names in javascript should be camelCase :)